spec/unit/resources/sh_cluster.rb

```diff
@@ -2,15 +2,15 @@
 
```

Comment:
For some reason that I haven't been able to figure out yet, this test doesn't get executed when you just run `rspec`.  The only way I've gotten it to run is if I do it explicitly via `rspec spec/unit/resources/sh_cluster.rb`.  So the test has been failing for a while, we just didn't notice it.

Reply:
I figured it out.  When you run `rspec` it only looks for files that end with `_spec.rb`.  The sh_cluster spec file was just named wrong, so it wasn't getting picked up.